pkg/api/validation/validation.go

```diff
@@ -3189,6 +3178,9 @@ func ValidateSecurityContext(sc *api.SecurityContext, fldPath *field.Path) field
 			allErrs = append(allErrs, field.Invalid(fldPath.Child("runAsUser"), *sc.RunAsUser, isNegativeErrorMsg))
 		}
 	}
+
+	// TODO when the versioned object is using feilds change this field path
+	allErrs = append(allErrs, ValidateSeccompProfile(sc.SeccompProfile, field.NewPath("pod", "metadata", "annotations", api.SeccompContainerAnnotationKeyPrefix+containerName))...)
```

Comment:
Not sure how big of an issue this is, but this field path will be slightly wrong for `PodTemplateSpec`s